tools/megahit_contig2fastg/megahit_contig2fastg.xml

```diff
@@ -1,8 +1,8 @@
-<?xml version='1.0' encoding='utf-8'?>
-<tool id="megahit_contig2fastg" name="MEGAHIT Contig2fastg" version="@VERSION@">
-    <description>for converting MEGAHIT's contigs to fastg</description>
+<?xml version='0.9' encoding='utf-8'?>
+<tool id="megahit_contig2fastg" name="megahit contig2fastg" version="@VERSION@">
+    <description>for converting MEGAHIT's contigs (.fa) to assembly graphs (.fastg)</description>
```

Comment:
what about `for converting MEGAHIT's contigs to assembly graphs`?

Reply:
I like that.